MAUI/Scheduler/reminders.md

```diff
@@ -0,0 +1,244 @@
+
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+Map those properties of the `Event` class with the [SfScheduler] control by using the [AppointmentMapping] and map `CustomReminder` properties with the [SchedulerReminder] by using [ReminderMapping.]
+
+{% tabs %}
+{% highlight xaml tabtitle="MainWindow.xaml" hl_lines="3 17 18 19 20" %}
+ <syncfusion:SfScheduler x:Name="Schedule" 
+                ItemsSource="{Binding Events}"
+                EnableReminder="True">
+            <syncfusion:SfScheduler.AppointmentMapping>
+                <syncfusion:AppointmentMapping
+                    Subject="EventName"
+                    StartTime="From"
+                    EndTime="To"
+                    AppointmentBackground="Color"
+                    IsAllDay="IsAllDay"
+                    StartTimeZone="StartTimeZone"
+                    EndTimeZone="EndTimeZone"
+                    RecurrenceExceptionDates="RecurrenceExceptions"
+                    RecurrenceRule="RecurrenceRule"
+                    RecurrenceId="RecurrenceId"
+                    Reminders="Reminders">
+                    <syncfusion:AppointmentMapping.ReminderMapping>
+                        <syncfusion:ReminderMapping IsDismissed="Dismissed"
+                                                    TimeBeforeStart="TimeBeforeStart"/>
+                    </syncfusion:AppointmentMapping.ReminderMapping>
+                </syncfusion:AppointmentMapping>
+            </syncfusion:SfScheduler.AppointmentMapping>
+        </syncfusion:SfScheduler>
+{% endhighlight %}
+{% highlight c# tabtitle="ReminderViewModel.cs" %}
+public class ReminderViewModel 
+{
+  ...
+  public ObservableCollection<Event> Events { get; set; } = new ObservableCollection<Event>();
+  this.Events.Add(new Event()
+  {
+    From = DateTime.Now,
+    To = DateTime.Now.AddHours(1),
+    Color = new SolidColorBrush((Color)ColorConverter.ConvertFromString("#FF339933")),
+    EventName = "Conference",
+    Reminders = new ObservableCollection<Reminder>
+    {
+        new Reminder { TimeBeforeStart = new TimeSpan(0)},
+    }
+    });
+}
+{% endhighlight %}
+{% endtabs %}
+
+## ReminderAlertOpening event
+Scheduler notify the appointment's reminder by [ReminderAlertOpening] event before the appointment's start time. The [ReminderAlertOpeningEventArgs] has the following property,
+* [Reminders]: Gets a list of reminders that are used to notify the appointment reminders.
+
+{% tabs %}
+{% highlight c# tabtitle="MainWindow.xaml.cs" hl_lines="1" %}
+scheduler.ReminderAlertOpening += Scheduler_ReminderAlertOpening;
+
+private void Scheduler_ReminderAlertOpening(object sender, ReminderAlertOpeningEventArgs e)
+{
+    var reminders = e.Reminders;
+    e.Reminders[0].IsDismissed = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Handling reminders for future appointment
+
+* If the Reminder alert time is less than the current time, then appointment reminder time will be calculated from the current time.
+
```

Comment:
Add toc file for main title
https://github.com/syncfusion-content/maui-docs/blob/development/maui-toc.html

Reply:
Added toc file for main title